Mixer e2e TestDenials is flaky

The test passed and failed in two successive release qualification runs.

Passed:
https://storage.googleapis.com/istio-prow/pull/istio-releases_daily-release/365/daily-e2e-rbac-auth/256/build-log.txt

Failed:
https://storage.googleapis.com/istio-prow/pull/istio-releases_daily-release/366/daily-e2e-rbac-auth/257/build-log.txt

I0201 09:56:38.184] --- FAIL: TestDenials (96.38s)
I0201 09:56:38.184] 	mixer_test.go:245: product page was not denied: could not retrieve product page in 1m0s: Last status: 200

--- a/tests/e2e/framework/kubernetes.go
+++ b/tests/e2e/framework/kubernetes.go
@@ -368,7 +368,7 @@ func (k *KubeInfo) generateIstio(src, dst string) error {
 		// Customize mixer's configStoreURL to limit watching resources in the testing namespace.
 		vs := url.Values{}
 		vs.Add("ns", *namespace)
-		content = replacePattern(k, content, "--configStoreURL=k8s://", "--configStoreURL=k8s://?%s"+vs.Encode())
+		content = replacePattern(k, content, "--configStoreURL=k8s://", "--configStoreURL=k8s://?"+vs.Encode())
 	}
 
 	// Replace long refresh delays with short ones for the sake of tests.

